components/support/migration/src/main/java/mozilla/components/support/migration/GeckoMigration.kt

```diff
@@ -4,10 +4,87 @@
 
 package mozilla.components.support.migration
 
+import android.util.AtomicFile
+import androidx.annotation.VisibleForTesting
+import mozilla.components.support.ktx.util.writeString
 import java.io.File
 import java.io.IOException
 
 private const val PREFS_FILE = "prefs.js"
+private const val PREFS_BACKUP_FILE = "prefs.js.backup.v"
+
+@VisibleForTesting
+internal var userPrefsToKeep = setOf("extensions.webextensions.uuids")
```

Comment:
const?

Reply:
Can't be a const because it's a set. Only primitives can be const :(.